src/LockIn-SignalRecovery_7265DSP/main.py

```diff
@@ -541,11 +538,11 @@ def wait_for_complete(self):
 
         starttime = time.time()
         while True:
-            # only the direct GPIB status byte call works as it can be acquired even when the lock-in is
-            # busy with auto sensitivity operation
             stb = self.port.port.read_stb()
-            if stb & 1 == 1:  # first byte indicates whether command is processed or not
+
+            if stb & 1 == 0:
                 break
+
             time.sleep(0.01)
             if time.time() - starttime > 20:
                 raise Exception("Timeout during wait for completion.")
```

Comment:
/implement Make this wait_for_complete method work like it's commonly done in the repo.